### PR TITLE
Issue 8189 - L2 caching example change

### PR DESCRIPTION
**Pull request by dineshvb**
## Purpose of this pull request

This pull request (PR) will cover #8189 

## Affected DevDocs pages

-  guides/v2.4/config-guide/cache/two-level-cache.html

### CODE REVIEW

File: src/guides/v2.4/config-guide/cache/two-level-cache.md [hunk 1]
```diff
@@ -21,28 +21,30 @@ Magento stores the hashed data version in Redis, with the suffix ':version' appe
 ```php
 'cache' => [
     [
```
Comment (rogyar) [inline src/guides/v2.4/config-guide/cache/two-level-cache.md:23]: We may still have an error here. The cache node should not contain another array. Please, correct me if I'm wrong
```suggestion

```

**Review by rogyar — CHANGES_REQUESTED** [review]

**Comment by dineshvb** [comment]
@rogyar Would need some technical assistance so that I could change it.

**Comment by rogyar** [comment]
Hi @dineshvb. Could you clarify, please, what type of technical assistance do you need? Thanks.

**Comment by dineshvb** [comment]
@rogyar just that array symbol, shall I delete it?

**Comment by rogyar** [comment]
Hi @dineshvb. Here is the correct version of your example 

```php
'cache' => [
            'frontend' => [
                'default' => [
                    'backend' => '\\Magento\\Framework\\Cache\\Backend\\RemoteSynchronizedCache',
                    'backend_options' => [
                        'remote_backend' => '\\Magento\\Framework\\Cache\\Backend\\Redis',
                        'remote_backend_options' => [
                            'persistent' => 0,
                            'server' => 'localhost',
                            'database' => '0',
                            'port' => '6370',
                            'password' => '',
                            'compress_data' => '1',
                        ],
                        'local_backend' => 'Cm_Cache_Backend_File',
                        'local_backend_options' => [
                            'cache_dir' => '/dev/shm/'
                        ]
                    ],
                    'frontend_options' => [
                        'write_control' => false,
                    ],
                    'use_stale_cache' => false,
                ]
            ]
        ],
        'type' => [
            'default' => ['frontend' => 'default'],
        ]
```

**Comment by dineshvb** [comment]
@rogyar @dobooth I have made the requested changes

**Comment by dobooth** [comment]
Is this another change for `master`?

**Comment by dineshvb** [comment]
> Is this another change for `master`?

No, these changes are for 2.4.2-develop branch since the issue only addresses the 2.4 version

**Comment by dobooth** [comment]
Thanks @dineshvb. To be clear, the `master` branch is for currently release docs, v2.3 and v2.4, which includes 2.4.1. If this is a fix to a live doc, then it would go to `master`. The `2.4.2-develop` branch is to document code for features that will be available in the 2.4.2 release, which is early next year. If we keep this as it is, it will not go live until the release next year.

**Comment by dineshvb** [comment]
@dobooth Rebased to master branch

**Comment by dobooth** [comment]
running tests

**Review by dobooth — APPROVED** [review]

**Pull request merged**